tests/core/test_backend_cache.py

```
@@ -313,7 +288,7 @@ def test_clear_cache_with_force_close(self) -> None:
                 mock_backend_class.return_value = mock_backend
 
                 # Acquire a pipeline
-                pipeline, key = acquire_pipeline(cfg)
+                _pipeline, key = acquire_pipeline(cfg)
```

Ruff (0.15.20)

[warning] Unpacked variable `key` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)